apps/condo/domains/resident/tasks/helpers/sendBillingReceiptNotifications.spec.js

```diff
@@ -18,9 +13,7 @@ describe('sendBillingReceiptNotifications', () => {
     describe('feature flag', () => {
         it('checks for proper result on enabled', async () => {
             setAllFeatureFlags(true)
-            const redisClient = getRedisClient()
-            await redisClient.set(REDIS_LAST_DATE_KEY, dayjs().toISOString())
-            expect(await sendBillingReceiptNotifications()).toBeUndefined()
+            expect(await sendBillingReceiptNotifications()).toEqual(undefined)
```

Comment:
If I'm correct you can use here `toBeUndefined()`
```suggestion
            expect(await sendBillingReceiptNotifications())..toBeUndefined()
```

Reply:
Yes you're right